spec/models/comment_spec.rb

```diff
@@ -14,7 +14,7 @@
 
     it "should be invalid if exactly duplicated" do
       expect(second_comment.valid?).to be_falsy
-      expect(second_comment.errors.keys).to include(:comment_content)
+      expect(second_comment.errors.attribute_names).to include(:comment_content)
```

Comment:
[ActiveModel::Errors::keys](https://api.rubyonrails.org/v6.1.7/classes/ActiveModel/Errors.html#method-i-keys) is deprecated, to be removed in Rails 7. I used the opportunity to switch to the suggested replacement.